app/code/Magento/InventoryInStorePickupApi/Api/Data/AddressInterface.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickupApi\Api\Data;
+
+/**
+ * Data interface for nearest pickup locations search request.
+ *
+ * @api
+ */
+interface AddressInterface
```

Comment:
I'd suggest adding extension attributes here. I'm not sure what happened with SourceSelection API that it has an Address interface which is not extendable but thats out of scope for this PR.

Reply:
I have created new issue just to do not miss anything: #2265.